packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -140,6 +140,17 @@ describe( 'Basic rendering', () => {
 		expect( searchInput ).toBeInTheDocument();
 	} );
 
+	it( 'should have aria-owns attribute to follow the ARIA 1.0 pattern', () => {
+		render( <LinkControl /> );
+
+		// Search Input UI.
+		const searchInput = screen.getByRole( 'combobox', { name: 'URL' } );
+
+		expect( searchInput ).toBeInTheDocument();
```

Comment:
Does it make sense to change this assertion to use `toBeVisible()`? I've found this to be a bit more specific query, since `toBeInTheDocument()` will be truthy if the element is not accessible to the user at all. 

```suggestion
		expect( searchInput ).toBeVisible();
```

Reply:
Makes sense. There are a few more occurrences of `toBeInTheDocument()`. Some of them use `not` and they actually test for some elements to not be in the DOM. I guess the other ones should be changed to `toBeVisible()`.